engines/tinsel/notebook.cpp

```diff
@@ -0,0 +1,64 @@
+/* ScummVM - Graphic Adventure Engine
```

Comment:
This file is Noir specific, so should go inside the noir subfolder.

Reply:
fixed